cpp/src/text/subword/data_normalizer.cu

```diff
@@ -262,11 +262,11 @@ __global__ void kernel_data_normalizer(unsigned char const* strings,
 
 }  // namespace
 
-data_normalizer::data_normalizer(rmm::cuda_stream_view stream, bool do_lower_case)
-  : do_lower_case(do_lower_case)
```

Comment:
I just realize that this is still a case of unused variable compiled without warning/error?

Reply:
Sorry, what variable was unused? 
The `do_lower_case` is still being used. The `stream` variable was being passed to the two removed utility methods.

Reply:
Ah I see. Sorry I missed that.